toolsHCK.ps1

```diff
@@ -907,11 +908,7 @@ function createprojecttarget {
                     $WntdDeviceFamily = $Manager.CreateDeviceFamily($HardwareIds[0], $HardwareIds)
                 }
                 $WntdTargetFamily = $WntdPI.CreateTargetFamily($WntdDeviceFamily)
-                $WntdPI.FindTargetFromDeviceFamily($WntdDeviceFamily) | foreach {
-                    if (($WntdTargetFamily.IsValidTarget($_)) -and ($_.Equals($toTarget))) {
-                        $WntdTargetFamily.CreateTarget($_) | Out-Null
-                    }
-                }
+                $WntdTargetFamily.CreateTarget($toTarget) | Out-Null

```

Comment:
Before, to prevent targeting identical targets on multiple machines with the createprojecttarget request on a specific machine's target, we iterated on all targets and ask if it is the wanted target and target it, since systems are more complicated, this iteration doesn't work, and the right way from the start is to call CreateTarget from the TargetFamily, Fixed that, but now we need the next commit which fixes the issue another more practical way and keep the SVVP support.